### PR TITLE
dockerize repository

### PR DESCRIPTION
**Pull request by MokkeMeguru**
Docker is the best solution for running codes in the machine of the lab or the company without changing its system.

an Example:
- docker-compose build
- docker-compose up -d
- docker-compose exec tensorflowtts /bin/bash
- tmux
- (any operation such as training)
- Ctrl-b d (exit from tmux)
- Ctrl-p Ctrl-q (exit from docker shell)

### CODE REVIEW

**Comment by MokkeMeguru** [comment]
ref. #56

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM tensorflow/tensorflow:2.3.0rc0-gpu
```
Comment (dathudeptrai) [inline dockerfile:1]: change it to tensorflow 2.2

File: dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+RUN apt-get update
+RUN apt-get install -y zsh tmux wget git
+RUN pip install git+https://github.com/dathudeptrai/TensorflowTTS.git
```
Comment (dathudeptrai) [inline dockerfile:4]: github.com/dathudeptrai/TensorflowTTS.git ->>> github.com/TensorSpeech/TensorflowTTS.git :D

**Comment by MokkeMeguru** [comment]
applied @dathudeptrai 's  two comments

**Comment by dathudeptrai** [comment]
do we need add some tutorial to README @MokkeMeguru

**Comment by MokkeMeguru** [comment]
I think no. We need end-to-end tutorial. In **its** tutorial, we need to use docker.
In other words, we don't have to pick up docker's tutorial.

**Pull request merged**